include/wx/dynarray.h

```diff
@@ -519,6 +524,8 @@ class wxBaseObjectArray : private wxBaseArray<T*>
         name(size_t n, Base::const_reference v) : Base(n, v) { }              \
         template <class InputIterator>                                        \
         name(InputIterator first, InputIterator last) : Base(first, last) { } \
+        template<typename U>                                                  \
+        name(std::initializer_list<U> list) : Base(list.begin(), list.end()) { } \
```

Comment:
This would have to use another helper macro, defined differently in C++11 and not case.